### ISSUE TITLE
No Option for Fingerprint or Face Unlock

### ISSUE THREAD

**Issue by FrenchToucan** [AChep/keyguard-app#577]
There are no options for biometric unlock although the description of the Security section in Settings indicates there would be.

I am on a Pixel 9 Android 14 September security patch. My Pixel 8 on Android 15 QPR1 Beta 1.1 does have a fingerprint option (as I recall).

**Comment by AChep** [comment]
Do you have a fingerprint added in the System settings?

**Comment by FrenchToucan** [comment]
> Do you have a fingerprint added in the System settings?

I do, yes.

**Comment by FrenchToucan** [comment]
When I installed Bitwarden and didn't have the fingerprint option, I knew something was up. And something was. I didn't have this option enabled. I swear I've never noticed it before and I've installed Android 14 probably 30 times across a variety of devices over the past year.

![Screenshot_20240927-230726.png](https://github.com/user-attachments/assets/a3e367bf-5483-42f3-a633-24ca2179cb0b)

So anyways, this issue can be closed.
